### PR TITLE
[WIP] completed japanese localisation for some files

### PR DESCRIPTION
**Pull request by Soukyuen**

### CODE REVIEW

**Comment by nanaya** [comment]
For generic pluralized translations - the ones with `:count` but without specific numbers (`{0}`, `[2,*]`, etc) - please only specify one.

So instead of

    '1 曲|:count 曲'

Just

    ':count曲'

(space separator optional)

**Comment by kj415j45** [comment]
(zh and zh-tw add space between ':count' and '曲', since the font issue)

File: resources/lang/ja/layout.php [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+            '503' => 'メンテナンス',
+        ],
+        'user' => [
+            '_' => 'ユーザー',
+            'getLogin' => 'ログイン',
+            'disabled' => '無効',
+
+            'register' => '登録',
+            'reset' => 'リセット',
+            'new' => '新',
```
[the remark this answers is not in the document]
Comment (Soukyuen) [reply resources/lang/ja/layout.php]: I actually tried to look for the context, but I couldn't figure out where this string appeared. It's basically a placeholder until I can find this :(

[the remark this answers is not in the document]
Comment (peppy) [reply resources/lang/ja/layout.php]: I can't readily check for context, but in that case 新しい probably works better than just 新

File: resources/lang/ja/admin.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+    'logs' => [
+        'index' => [
+            'title' => 'ログビューワー',
+        ],
+    ],
+
+    'pages' => [
+        'root' => [
+            'title' => 'アドミンのコンソールっぽいの',
```
[the remark this answers is not in the document]
Comment (nanaya) [reply resources/lang/ja/admin.php]: ...lol what have I done. Shouldn't have put that `thingy` word in English version. Something like "管理コンソール" should be fine.

[the remark this answers is not in the document]
Comment (Soukyuen) [reply resources/lang/ja/admin.php]: it's cute but yea will do

File: resources/lang/ja/layout.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (notbakaneko) [reply resources/lang/ja/layout.php]: I think most of the `user` ones are actually unused

**Comment by peppy** [comment]
@Soukyuen any chance you can apply the review changes so we can get this basic level of localisation live?

**Comment by Soukyuen** [comment]
oh what the heck

**Comment by Soukyuen** [comment]
@peppy it's updated if you wanted me to let you know

**Comment by peppy** [comment]
Let's get this up and running!

**Pull request merged**